IndustrialUnit.WpfUI/ViewModels/EquipmentViewModel.cs

```diff
@@ -0,0 +1,165 @@
+﻿using IndustrialUnit.WpfUI.Models;
+using IndustrialUnitDatabase;
+using System;
+using System.Collections.ObjectModel;
+using System.IO;
+using System.Windows.Input;
+
+namespace IndustrialUnit.WpfUI.ViewModels
+{
+  public class EquipmentViewModel : BaseViewModel
+  {
+    private ObservableCollection<EquipmentViewProperties> _equipments;
+    public ObservableCollection<EquipmentViewProperties> Equipments
+    {
+      get
+      {
+        return _equipments;
+      }
+      set
+      {
+        _equipments = value;
+        OnPropertyChanged();
+        SelectedEquipment = new EquipmentViewProperties();
+      }
+    }
+
+    private EquipmentViewProperties _selectedEquipment;
+    public EquipmentViewProperties SelectedEquipment
+    {
+      get
+      {
+        return _selectedEquipment;
+      }
+      set
+      {
+        _selectedEquipment = value;
+        OnPropertyChanged();
+      }
+    }
+
+    private string _messageToView;
+    public string MessageToView
+    {
+      get
+      {
+        return _messageToView;
+      }
+      set
+      {
+        _messageToView = value;
+        OnPropertyChanged();
+      }
+    }
+
+
+    public static ObservableCollection<EquipmentViewProperties> GetEquipments()
+    {
+      ObservableCollection<EquipmentViewProperties> equipmentCollection = new();
+      var getEquipment = SQLiteDataAccess.GetAll("Equipment");
+
+      var rowNumber = getEquipment.Rows.Count;
+
+      for (int i = 0; i < rowNumber; i++)
+      {
+        var item = getEquipment.Rows[i];
+        equipmentCollection.Add(
+          new EquipmentViewProperties()
+          {
+            Id = Convert.ToInt32(item.ItemArray[0]),
+            ItemType = Convert.ToString(item.ItemArray[1]),
+            Capacity = Convert.ToDecimal(item.ItemArray[2]),
+            Pressure = Convert.ToDecimal(item.ItemArray[3]),
+            PowerConsumption = Convert.ToDecimal(item.ItemArray[4]),
+            Manufacturer = Convert.ToString(item.ItemArray[5]),
+            Model = Convert.ToString(item.ItemArray[6]),
+            UnitPrice = Convert.ToDecimal(item.ItemArray[7]),
+          });
+      }
+      return equipmentCollection;
+    }
+
+    public ObservableCollection<EquipmentViewProperties> GetFilteredEquipments(ObservableCollection<EquipmentViewProperties> Equipments)
```

Comment:
Ez a paraméter elnevezés egy jó prank lenne, mivel egy létező property-t ugyanígy hívnak. Nevezd át `equipments`-re, meg úgy általában tartsd be az elnevezési konvenciókat, különben te magad se fogod érteni ezt fél év múlva.